code/datums/sound_system.dm

```diff
@@ -0,0 +1,34 @@
+/*
+This list is used because using the environment var (and its presets) changes the sound environment
+for ALL sounds, including sounds already playing. This means that if you play a sound without reverb, any
+sound that is already playing will have its reverb removed. The workaround is to use the echo var. Echo
+works on one sound only. However, echo does not have presets. This list stores the arguments to be passed
+to echo, in order to simulate an environment preset.
+*/
+/datum/sound/var/list/presets=list(\
+	list(7.5,   1.00,   -1000,  -100,   0,  1.49,  0.83, 1.0,  -2602, 0.007,  200,   0.011, 0.250,  0.00,  0.25, 0.000, -5.0,  5000.0),

```

Comment:
please add comments with what echo type these are to each of these, or maybe even merge it with the guide?